vault/identity_store_test.go

```diff
@@ -122,9 +139,9 @@ func TestIdentityStore_UnsealingWhenConflictingAliasNames(t *testing.T) {
 	alias2 := &identity.Alias{
 		ID:             "alias2",
 		CanonicalID:    "entity2",
-		MountType:      "github",
-		MountAccessor:  meGH.Accessor,
-		Name:           "GITHUBUSER",
+		MountType:      "userpass",
+		MountAccessor:  meUP.Accessor,
+		Name:           "USERPASSUSER",
```

Comment:
Isn't the point of this test to have two aliases with the same accessor but names in different cases?

Reply:
Oh, good point. I think I updated this test before I updated the automated merge behaviour. I've reverted this change.